src/test/scala/squid/classembed/ClassEmbeddingTests.scala

```diff
@@ -33,40 +33,64 @@ class ClassEmbeddingTests extends MyFunSuite {
   test("Basics") {
     
     Emb.Object.Defs.foo.value eqt code"(arg: Int) => MyClass.foo(arg.toLong).toInt"
-    
```

Comment:
Please configure your editor _not_ to remove indentation spaces, and rebase the commits to remove these changes.